os/net/packetbuf.h

```diff
@@ -268,7 +268,7 @@ const linkaddr_t *packetbuf_addr(uint8_t type);
  * \brief      Checks whether the current packet is a broadcast.
  * \retval 0   iff current packet is not a broadcast
```

Comment:
Documentation should also be updated if the return types are changed.

Reply:
good proposal